### PR TITLE
fix: install epel repo in RHEL 7 using static repo file

### PR DESCRIPTION
**Pull request by supershal**
**What problem does this PR solve?**:
This PR is stacked on https://github.com/mesosphere/konvoy-image-builder/pull/781
Installs epel release for RHEL/Centos 7.9 using static repo files

### CODE REVIEW

**Comment by supershal** [comment]
Tested the fix locally successfully on Centos 7 AMI

File: ansible/roles/repo/tasks/redhat.yaml [hunk 1]
```diff
@@ -99,21 +99,12 @@
...
     - ansible_distribution_major_version == '8'
 
-- name: add epel gpg key for centos 7
-  rpm_key:
-    state: present
-    key: "{{ epel_centos_7_rpm_gpg_key }}"
-  when:
-    - ansible_distribution_major_version == '7'
-
 - name: install epel-release for centos 7
```
Comment (dkoshkin) [inline ansible/roles/repo/tasks/redhat.yaml:102]:
```suggestion
- name: add EPEL rpm repository
```

File: ansible/roles/repo/tasks/redhat.yaml [hunk 2]
```diff
@@ -99,21 +99,12 @@
...
-  yum:
```
Comment (dkoshkin) [inline ansible/roles/repo/tasks/redhat.yaml:110]: Wow I didn't realize that this just adds some repos 😬

**Review by dkoshkin — APPROVED** [review]

**Comment by supershal** [comment]
CI job: https://github.com/mesosphere/konvoy-image-builder/actions/runs/4746923145

File: ansible/roles/repo/files/etc/yum.repos.d/epel-7.repo [hunk 3]
```diff
@@ -0,0 +1,26 @@
+[epel]
```
[the remark this answers is not in the document]
Comment (supershal) [reply ansible/roles/repo/files/etc/yum.repos.d/epel-7.repo:1]: probably not. these files uses `https://mirrors.fedoraproject.org` which is used by all the Centos versions

**Pull request merged**